Issue https://github.com/h6ah4i/android-advancedrecyclerview/issues/1…

thefex opened the pull request:
…42 fix

h6ah4i commented on library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/utils/BaseWrapperAdapter.java:
```diff
@@ -121,7 +133,7 @@ protected void onHandleWrappedAdapterItemRangeChanged(int positionStart, int ite
     }
 
     protected void onHandleWrappedAdapterItemRangeChanged(int positionStart, int itemCount, Object payload) {
-        notifyItemRangeChanged(positionStart, itemCount, payload);
```
@thefex Why did you modified this line?

thefex commented:
Reverted change in this method, my fault - sorry I have written fix for old version for library (which didn't had notifyItemRangeChanged signature with Object) - then I updated source and forgot to fix that.

h6ah4i commented:
Thank you for fixing the `notifyItemRangeChanged()` method. It looks good to me ; )

I am going to merge it manually, because I want to merge it into the `develop` branch.

h6ah4i commented:
Done. It will be included in v0.8.4. Thanks!
